Mathlib/Tactic/Widget/Calc.lean

```diff
@@ -116,7 +116,7 @@ def suggestSteps (pos : Array Lean.SubExpr.GoalsLocation) (goalType : Expr) (par
 @[server_rpc_method]
 def CalcPanel.rpc := mkSelectionPanelRPC suggestSteps
   "Please select subterms."
-  "Calc 🔍"
+  "Calc 🔍️"
```

Comment:
These are also intended to be non-breaking; I think using the unicode escape would be appropriate here.

Reply:
same here, there is no space change, but the magnifying glass received a emoji-variant-selector because its specified in the list of emojis.

Reply:
For context, there are currently no non-breaking spaces in any parts of mathlib except in docstrings about condensed mathematics & cat. theory because Dagur's IDE automatically adds them after each closing \`

And the dependency PRs fixing whitespace are mainly because Dagur's quite busy and commiting a lot :D